test/support/builders/CISDeductionsBuilder.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package support.builders
+
+import models.get.CISDeductions
+import support.builders.GetPeriodDataBuilder.aGetPeriodData
+import support.utils.TaxYearUtils.{taxYear, taxYearEOY}
+
+object CISDeductionsBuilder {
+
+  val aCISDeductions = CISDeductions(
+    fromDate = s"$taxYearEOY-04-06",
+    toDate = s"$taxYear-04-05",
+    contractorName = Some("Contractor 1"),
```

Comment:
Try to use aCISSubmission.contractorName. The following gives more information as to why.